### PR TITLE
fix example in ads docs

### PR DESCRIPTION
**Pull request by mingkang111**
# Description
This PR aims to fix the example in ads docs.
* `%use_session` in dataflow-spark-magic
* `Quick start` example of using PyTorch Framework.

### CODE REVIEW

**Review by mayoor — APPROVED** [review]
Looks good to me

File: .github/workflows/codeql.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# For most projects, this workflow file will not need changing; you simply need
```
Comment (jasper-schneider) [inline .github/workflows/codeql.yml:1]: This file looks like it's setting up some sort of post commit hook or something?
It this meant to be part of this PR? It doesn't seem related to the example updates.

**Review by jasper-schneider — APPROVED** [review]

File: .github/workflows/codeql.yml (hunk 1 above)
Comment (mingkang111) [reply .github/workflows/codeql.yml:1]: I didn’t add this file when I do git add and git commit. It was autogenerated. The first two lines of that file say that it just needs to be committed into my repository. I created this branch from the main branch. Should I try to remove it?

Comment (jasper-schneider) [reply .github/workflows/codeql.yml:1]: The main branch has this file, but none of the release branches do.
I don't have a ton of experience with github workflows, so I don't know what the consequences are of this file right now.
Did you create your branch off main or off the 2.7.x branch that you're targeting? From the commit list, it looks like you're pulling in some changes to the main branch on top of the changes you directly made to the docs.

Comment (mingkang111) [reply .github/workflows/codeql.yml:1]: I created the branch off main branch. But now targeting 2.7.x. I think that is the issues comes from. Let me fix it.
